cmd/export.go

```diff
@@ -36,52 +36,61 @@ func init() {
 	Marcel.AddCommand(cmd)
 
 	var usersWPassword bool
+	var pretty bool
 
 	var users = &cobra.Command{
 		Use:   "users [FILE]",
 		Short: "Exports users from marcel's database",
 		Args:  cobra.MaximumNArgs(1),
 
 		RunE: func(_ *cobra.Command, _ []string) error {
-			return export.Users(usersWPassword, exportFile)
+			return export.Users(pretty, usersWPassword, exportFile)
 		},
 	}
 
 	users.Flags().BoolVar(&usersWPassword, "withPassword", false, "Include each user's password")
+	users.Flags().BoolVar(&pretty, "pretty", false, "Indent json export")
```

Comment:
Since pretty is declared on all export subcommands, you can use a persistent flag on the export command itself.

Reply:
Well, yes !